src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -523,15 +524,15 @@ public static <T> List<LfNetwork> load(T network, LfNetworkLoader<T> networkLoad
         return load(network, networkLoader, parameters, Reporter.NO_OP);
     }
 
-    public static <T> List<LfNetwork> load(T network, LfNetworkLoader<T> networkLoader, SlackBusSelector slackBusSelector, Reporter reporter) {
```

Comment:
This removal is okay for me, to be confirmed by @geofjamg

Reply:
yes